ts/Core/Chart/Chart.ts

```diff
@@ -477,6 +477,27 @@ class Chart {
             this.callback = callback;
             this.isResizing = 0;
 
+            const zooming = optionsChart.zooming;
+
+            if (zooming) {
+                optionsChart.pinchType = pick(
+                    zooming.pinchType,
+                    optionsChart.pinchType
+                );
+                optionsChart.zoomBySingleTouch = pick(
+                    zooming.singleTouch,
+                    optionsChart.zoomBySingleTouch
+                );
+                optionsChart.zoomType = pick(
+                    zooming.type,
+                    optionsChart.zoomType
+                );
+                optionsChart.zoomKey = pick(
+                    zooming.zoomKey,
```

Comment:
`zooming.zoomKey` is a bit repetetive isn't it? I think it should be `zooming.key`.